src/composables.ts

```diff
@@ -16,6 +16,11 @@ import type {
 
 import type { PasswordRequestParams, Source } from './types'
 
+function isDocument(doc: MaybeRef<Source>) {
+  const docValue = toValue(doc)
```

Comment:
I wouldn't do the cast here, but in the function invocation (`onBeforeUnmount` hook)

And please move `isDocument` to `src/utils.ts`

Reply:
I rebased my branch onto `develop` and implemented your feedback.

`toValue` doesn't resolve `ShallowRef` types, that's why I did the extra cast in the function.
The more I think about it, I think this is a bug in the vue.js types. But I'm not quite sure yet.
Note that calls of the `isDocument` yield a Typescript error now, but the code works just fine.